freud/density.pyx

```diff
@@ -228,17 +228,33 @@ cdef class GaussianDensity(_Compute):
         """:class:`freud.box.Box`: Box used in the calculation."""
         return freud.box.BoxFromCPP(self.thisptr.getBox())
 
-    def compute(self, system):
+    def compute(self, system, values=None):
         R"""Calculates the Gaussian blur for the specified points.
 
         Args:
             system:
                 Any object that is a valid argument to
                 :class:`freud.locality.NeighborQuery.from_system`.
+            values ((:math:`N_{points}`) :class:`numpy.ndarray`):
+                Values associated with the system points used to calculate the
+                convolution. Calculates Gaussian blur if :code:`None`.
+                (Default value = :code:`None`).
```

Comment:
Mention that the "Gaussian blur" is equivalent to every point having a value of 1.
```suggestion
                convolution. Calculates Gaussian blur (equivalent to providing
                a value of 1 for every point) if :code:`None`. (Default value
                = :code:`None`).
```